woocommerce/woocommerce-android
# Feature/7863 name picker step

Closes: #7863 


### Description
Adds step to set the store name. 

### Testing instructions


Kick of store creation
See "Coninue"  button is disabled. 
Type any name and click continue to navigate to the next step -> Domain picker

Note: domain picker doesn't prefill the domain textfield with the store name yet. It will be done in a[ different PR](https://github.com/woocommerce/woocommerce-android/pull/7879)

### Images/gif


https://user-images.githubusercontent.com/2663464/202232680-b4db9286-2735-44ec-92e1-dd4b12033d1c.mp4
